src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Get.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Json
+{
+    /// <summary>
+    /// Contains the extensions methods for using JSON as the content-type in HttpClient.
+    /// </summary>
+    public static partial class HttpClientJsonExtensions
+    {
+        public static Task<object?> GetFromJsonAsync(this HttpClient client, string requestUri, Type type, JsonSerializerOptions? options = null, CancellationToken cancellationToken = default)
```

Comment:
Per my earlier comment, I think `string requestUri` should be `string? requestUri`.  Similarly below `Uri requestUri` should be `Uri? requestUri`.